packages/manager/apps/web/client/app/email-domain/email/redirection/email-domain-email-redirection.controller.js

```diff
@@ -1,84 +1,116 @@
 import map from 'lodash/map';
 
-angular.module('App').controller(
-  'EmailDomainEmailRedirectionCtrl',
-  class EmailDomainEmailRedirectionCtrl {
-    /**
-     * Constructor
-     * @param $scope
-     * @param $stateParams
-     * @param $q
-     * @param $translate
-     * @param Alerter
-     * @param WucEmails
-     */
-    constructor($scope, $stateParams, $q, $translate, Alerter, WucEmails) {
-      this.$scope = $scope;
-      this.$stateParams = $stateParams;
-      this.$q = $q;
-      this.$translate = $translate;
-      this.Alerter = Alerter;
-      this.WucEmails = WucEmails;
-    }
+angular
+  .module('App')
+  .constant('PAGE_SIZE_STORAGE_KEY', 'REDIRECTION_EMAILS_PAGINATION')
+  .controller(
+    'EmailDomainEmailRedirectionCtrl',
+    class EmailDomainEmailRedirectionCtrl {
+      /**
+       * Constructor
+       * @param $scope
+       * @param $stateParams
+       * @param $q
+       * @param $translate
+       * @param $window
+       * @param Alerter
+       * @param WucEmails
+       * @param PAGE_SIZE_STORAGE_KEY
+       */
+      constructor(
+        $scope,
+        $stateParams,
+        $q,
+        $translate,
+        $window,
+        Alerter,
+        WucEmails,
+        PAGE_SIZE_STORAGE_KEY,
+      ) {
+        this.$scope = $scope;
+        this.$stateParams = $stateParams;
+        this.$q = $q;
+        this.$translate = $translate;
+        this.$window = $window;
+        this.Alerter = Alerter;
+        this.WucEmails = WucEmails;
+        this.$scope.PAGE_SIZE_STORAGE_KEY = PAGE_SIZE_STORAGE_KEY;
```

Comment:
Not need to attach it to the `$scope`.
You can directly refer to `PAGE_SIZE_STORAGE_KEY`.